blob/ChangeLog.md

```diff
@@ -1,3 +1,6 @@
+2018.1 - version 1.1.0
```

Comment:
Is this date correct?

Reply:
Not accurate. I will update since I will have more fixes in this release.